zerver/management/commands/runtusd.py

```diff
@@ -0,0 +1,57 @@
+import os
+from typing import Any
+from urllib.parse import parse_qs, urlencode, urlsplit, urlunsplit
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandParser
+
+from scripts.lib.zulip_tools import run
+
+
+def set_query_parameter(url: str, param_name: str, param_value: str) -> str:
```

Comment:
Maybe we can move this to `scripts.lib.zulip_tools`.

Reply:
No

Reply:
I think given that, this is probably a fine home for it, or maybe `zerver/lib/tusd` if we want the function to be more readily unit tested.